packages/ui/src/ui/store/selectors/groups.ts

```diff
@@ -127,33 +140,34 @@ export const getGroupsFlattenTree = createSelector([getGroupsTreeFilteredAndSort
 });
 
 // Editor
-export const getGroupEditorData = (state) => state.groups.editor;
-export const getGroupEditorVisible = (state) => state.groups.editor.groupName.length > 0;
-export const getGroupEditorGroupName = (state) => state.groups.editor.groupName;
+export const getGroupEditorData = (state: RootState) => state.groups.editor;
+export const getGroupEditorVisible = (state: RootState) => state.groups.editor.groupName.length > 0;
+export const getGroupEditorGroupName = (state: RootState) => state.groups.editor.groupName;
 // eslint-disable-next-line camelcase
-export const getGroupEditorGroupIdm = (state) =>
-    flags.get(state.groups.editor.data.$attributes?.upravlyator_managed);
-const getGroupEditorIdmData = (state) => state.groups.editor.idmData;
-export const getGroupEditorIdmDataVersion = (state) => state.groups.editor.idmData.version;
-export const getGroupEditorIdmDataOtherMembers = (state) =>
+export const getGroupEditorGroupIdm = (state: RootState) =>
+    flags.get(state.groups.editor.data.$attributes?.upravlyator_managed as FlagType);
+const getGroupEditorIdmData = (state: RootState) => state.groups.editor.idmData;
+export const getGroupEditorIdmDataVersion = (state: RootState) =>
+    state.groups.editor.idmData.version;
+export const getGroupEditorIdmDataOtherMembers = (state: RootState) =>
     state.groups.editor.idmData.group.other_members;
 
 export const getGroupEditorSubjects = createSelector([getGroupEditorIdmData], (idmData) => {
     const {
-        group: {members, responsible},
+        group: {members, responsibles},
```

Comment:
That typo was here all along. 

I suspect that idm and group editor were never tested in opensource yt.